go.mod

```diff
@@ -7,13 +7,10 @@ go 1.19
 // Also, use our fork with fixes for unimplemented methods (required for Go 1.16).
 replace github.com/denisenkom/go-mssqldb => github.com/grafana/go-mssqldb v0.9.2
 
-// Avoid using v2.0.0+incompatible Redigo used by dependencies as the latest maintained branch of Redigo is v1.
```

Comment:
Are all the changes in this file related to the upgrade, even the lines that don't mention docker such as this one?

Reply:
No, that is just an unused line.